notebooks/time-to-merge-prediction/workshop/00_read_env.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Let us not print this as well. Not sure whats a good way to verify that the token is right. Will have to check</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>